### ISSUE TITLE
Unlocking root throws exception on node change to root level

### ISSUE THREAD

**Issue by tanczosm** [marcantondahmen/sortable-tree#6]
On the demo page: [https://marcantondahmen.github.io/sortable-tree/#unlocking-root](https://marcantondahmen.github.io/sortable-tree/#unlocking-root)

If you drag any node to root level it generates an uncaught exception "Cannot read properties of null".

I thought it was on my end but I'm getting the same issue on the demo page.

```
demo.js?t=1691690480035:1  Uncaught (in promise) TypeError: Cannot read properties of null (reading 'collapse')
    at _.onDrop (demo.js?t=1691690480035:1:9014)
    at s (demo.js?t=1691690480035:1:7013)
    at s.next (<anonymous>)
    at a (demo.js?t=1691690480035:1:7177)
```

**Comment by marcantondahmen** [comment]
Hi, thanks for reporting. I just released 0.7.1 in order to fix it.

**Comment by tanczosm** [comment]
Thanks!
